hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMultiTargetMessageTask.java

```diff
@@ -56,7 +54,6 @@ protected void processMessage() throws Throwable {
         MultiTargetCallback callback = new MultiTargetCallback(targets);
         for (Address target : targets) {
             Operation op = operationFactory.createOperation();
-            op.setCallerUuid(endpoint.getUuid());
```

Comment:
It is not the case it not needed. It is wrong to set that uuid. Endpoint.getUuid is the uuid of client. Caller uuid is used only in one place which is AuthBaseMessageTask(this is true for only master). In maintenance this uuid is not used at all anywhere. And for all the operations generated from client only this one sets the uuid of client. I removed it to make it same for all operations and same with master.